src/frontend/utils/periode.ts

```diff
@@ -21,7 +21,7 @@ export const validerPeriode = (periode: Periode): undefined | Partial<Periode> =
         return { tom: 'Mangler tildato for periode' };
     }
 
-    if (!erDatoEtterEllerLik(periode.fom, periode.tom)) {
+    if (!erDatoEtterEllerLik(periode.tom, periode.fom)) {
```

Comment:
Burde denne funksjonen ta inn en periode i stedet for tom og fom sånn her? Er ikke så intuitivt hva som skal være først nødvendigvis

Reply:
Det er på en måte det denne metoden gjør `validerPeriode` :) 
Den kaller videre en støttemetode som har
`export const erDatoEtterEllerLik = (fra: string, til: string): boolean => {`
![image](https://github.com/navikt/tilleggsstonader-sak-frontend/assets/937168/34972841-aaa1-414c-9206-198347c5185a)

Det som egentlige var feil her var at det var feil i `erDatoEtterEllerLik`, som nå er fikset i https://github.com/navikt/tilleggsstonader-sak-frontend/pull/147/commits/0c735e44bedea9a30c7112eabd7bf85dc2a73436